$OpenBSD$

Fix "undefined symbols" when racket loads libGL and libGLU. Fixed in 5.3.3.8.

--- collects/sgl/gl.rkt.orig	Wed Mar  6 01:58:29 2013
+++ collects/sgl/gl.rkt	Wed Mar  6 02:00:28 2013
@@ -11,11 +11,11 @@
 (define gl-lib (case stype
                  [(windows) (ffi-lib "opengl32")]
                  [(macosx) (ffi-lib "/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL")]
-                 [else (ffi-lib "libGL" '("1" ""))]))
+                 [else (ffi-lib "libGL" '("1" "") #:global? #t)]))
 (define glu-lib (case stype
                   [(windows) (ffi-lib "glu32")]
                   [(macosx) (ffi-lib "/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGLU")]
-                  [else (ffi-lib "libGLU" '("1" ""))]))
+                  [else (ffi-lib "libGLU" '("1" "") #:global? #t)]))
 
 (define win32?
   (and (eq? 'windows stype)
